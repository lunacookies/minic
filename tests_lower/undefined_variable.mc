func a {
	var x = y;
}
