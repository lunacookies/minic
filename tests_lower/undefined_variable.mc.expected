func a
	var x void
	{
		set x = <missing>
	}
tests_lower:18..19: error: undefined variable
